source/_posts/2019-10-25-rip-hassbian.markdown

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "R.I.P Hassbian"
+description: "The end of Hassbian"
+date: 2019-10-25 01:11:03
+date_formatted: "October 25, 2019"
+author: Fredrik Lindqvist
+author_twitter: Landrash
+comments: true
+categories: community
+---
+
+It's time for the part of Home Assistant that once was the simplest way to install Home Assistant to retire. Hassbian is being retired.
+
+## R.I.P Hassbian
+
+Some off you will probably wonder why Hassbian is being retired and I'll try to give a proper motivation. 
+First of is the age old factor of **time**. I as the lead developer of Hassbian hasn't been able to give it the time and attention it requires and there has been few others that have pushed the project further. The one exception being [@ludeeus] who's done most of the heavy lifting when the image was retooled to use a proper apt package and repository.
+Second is that it's no longer the best option for most to use as an installation method. Hass.io has surpassed Hassbian in almost all ways I can imagine.
+
+## Next Step for Hassbian
+Since Hassbian has been around for quite a while there's quite a few users that don't want Hassbian to go away. To make this as easy as possible for those users here's the plan.
```

Comment:
```suggestion
Since Hassbian has been around for quite a while, there are quite a few users that don't want Hassbian to go away. To make this as easy as possible for those users, here's the plan.
```